src/comfystream/client.py

```diff
@@ -117,7 +131,10 @@ async def get_audio_output(self):
         return await tensor_cache.audio_outputs.get()
     
     async def get_text_output(self):
-        return await tensor_cache.text_outputs.get()
+        try:
+            return tensor_cache.text_outputs.get_nowait()
+        except:
+            return None
```

Comment:
We should add a case where we log the error ( if its not just an empty queue ), e.g. :

```suggestion
    except asyncio.QueueEmpty:
        # Expected case - queue is empty, no text available
        return None
    except Exception as e:
        # Unexpected errors logged for debugging
        logger.warning(f"Unexpected error in get_text_output: {e}")
        return None
```